source/common/filter/config_discovery_impl.cc

```diff
@@ -232,41 +230,19 @@ void FilterConfigProviderManagerImplBase::applyLastOrDefaultConfig(
   }
 }
 
-std::tuple<ProtobufTypes::MessagePtr, std::string> HttpFilterConfigProviderManagerImpl::getMessage(
-    const envoy::config::core::v3::TypedExtensionConfig& filter_config,
-    Server::Configuration::ServerFactoryContext& factory_context) const {
-  auto& factory =
-      Config::Utility::getAndCheckFactory<Server::Configuration::NamedHttpFilterConfigFactory>(
-          filter_config);
-  ProtobufTypes::MessagePtr message = Config::Utility::translateAnyToFactoryConfig(
-      filter_config.typed_config(),
-      factory_context.messageValidationContext().dynamicValidationVisitor(), factory);
-  return {std::move(message), factory.name()};
-}
-
-absl::string_view HttpFilterConfigProviderManagerImpl::statPrefix() const { return HttpStatPrefix; }
-
-ProtobufTypes::MessagePtr HttpFilterConfigProviderManagerImpl::getDefaultConfig(
-    const ProtobufWkt::Any& proto_config, const std::string& filter_config_name,
-    Server::Configuration::FactoryContext& factory_context, bool last_filter_in_filter_chain,
-    const std::string& filter_chain_type,
-    const absl::flat_hash_set<std::string>& require_type_urls) const {
-  auto* default_factory =
-      Config::Utility::getFactoryByType<Server::Configuration::NamedHttpFilterConfigFactory>(
-          proto_config);
-  if (default_factory == nullptr) {
+void FilterConfigProviderManagerImplBase::validateProtoConfigDefaultFactory(
```

Comment:
IIUC both this method and `validateProtoConfigTypeUrl` are only called once.
Is there a plan to call them in other places in later PRs? If not, then we should just inline their contents in the call place.